nextcord/types/role.py

```diff
@@ -21,6 +22,7 @@ class Role(TypedDict):
     tags: NotRequired[RoleTags]
     unicode_emoji: NotRequired[str]
     icon: NotRequired[str]
+    flags: RoleFlags
```

Comment:
This isn't going to work because the type of this parameter in the raw payload is just a simple integer. These typed dicts are supposed to represent the raw payloads, after all.